open-sphere-base/analysis/src/main/java/io/opensphere/analysis/table/model/SpreadsheetUtilities.java

```diff
@@ -0,0 +1,42 @@
+package io.opensphere.analysis.table.model;
+
+import javax.swing.JTable;
+
+/**
+ * Spreadsheet-related utilities for table models.
+ */
+public class SpreadsheetUtilities
```

Comment:
Utility classes (those with all static methods) should be final with a private no-op default constructor.